### ISSUE TITLE
Best "loss" acheived

### ISSUE THREAD

**Issue by ryanmjacobs** [minimaxir/textgenrnn#125]
I've been doing a grid search on most of hyperparameters provided, and have reduced my loss to 0.529. My dataset is 2.9 MB (roughly 102,000 lines of text).

I would love to hear some anecdotal evidence as to what loss values are achievable and/or realistic with this framework.

**Comment by jurkstas** [comment]
I got 0.0697 training on poem The Seasons by Kristijonas Donelaitis. 1211 sentences, 141 445 symbols.
`textgen.train_from_file('donelaitis.txt', num_epochs=100,gen_epochs = 10,batch_size = 256)`
